site/docs/v1/tech/apis/form-fields/_form-field-control.adoc

```diff
@@ -8,5 +8,9 @@ The Form Field control. The possible values are:
  * `textarea`
  * `text`
 +
-When a predefined [field]#key# field is used, [field]#control# will be defaulted. The [field]#key# value of +
-`user.password` defaults to `password`. All other [field]#key# values default to `text`.
\ No newline at end of file
++
+Defaults can change depending on the type of `key`. The defaults are:
++
+* `select` if key is any of the following: `registration.preferredLanguages`, `registration.timezone`, `user.preferredLanguages`, `user.timezone`
+* `checkbox` if key is any of the following: `registration.roles`, `user.twoFactorEnabled`
+* `text` in all other cases
```

Comment:
Review this rendered on the page, it doesn't look quite right. There is no space between the end of the above list and this one. Be sure to do a visual walk through of all doc changes to check for formatting issues.